### DISCUSSION TITLE
🛠️ Project ideas to Build with Bandada

### DISCUSSION THREAD

**Discussion by andyguzmaneth** [privacy-scaling-explorations/bandada#367] [Ideas]
### ⚙️ Protocol Improvements
- Write custom connector for Lens, Farcaster
- Write custom connector that generates Semaphores identities based on ENS
- Create new custom credential providers (web2 and web3)
    - Web3 - Polygon ID attestations
    - Web3 - The Badge attestations
    - Web3 - POAPs
    - Web3 - zkSBTs (soul-bound-tokens)
    - Web3 - Coinbase ID attestations
    - Web3 - Optimism attestations
    - Web3 - Verax
    - Web2 - Reddit reputation
    - Web2 - Youtube followers  

### 🧩 Integrations 
- Integrations into the AragonDAO tooling ecosystem
- Integrate [RLN](https://rate-limiting-nullifier.github.io/rln-docs/) with Bandada to give an additional option to create spam-protected groups
- Integrating Bandada with Zupass
- Integrating Bandada with TLSNotary
- Integrating Bandada with Gitcoin Passport
- Integrating Bandada with zkEmail
- Farcaster frames.

### 📱 Apps Ideas
- DeFi - Backend of an Anonymous Exchange
- DAOs - Private forum that only people with X can enter
- Airdrops - Anon 
- DAO - Recognition system / tiered access based on GitHub contributions to specific repos
- Web3 - Event management system for organising and managing private events / meet-ups (invitations, RSVPs and participant identities are anonymous).
- Web2 - Private Peer Support Networks, join groups based on various topics ensuring complete anonymity.

### 🧪 Infrastructure
- Public Bandada group registry with indexing (group properties as hashtags/scope/tags).
- Extend customisation of groups (beyond name and description).

**Comment by ghost** [comment]
I am building a proof of profit defi trading protocol so traders can prove their profitability and receive capital. 
Idea - A private dex for only profitable traders?
